src/WebJobs.Script/Workers/WorkerConcurrencyManager.cs

```diff
@@ -318,6 +317,26 @@ internal bool IsEnoughMemoryToScale(long hostProcessSize, IEnumerable<long> work
             return true;
         }
 
+        internal bool CanScale(IEnumerable<IRpcWorkerChannel> workerChannels)
```

Comment:
Consider adding a method summary to explain _why_ we are doing _what_ we are doing inside this method, to help future readers.